en/components/grid_toolbar.md

```diff
@@ -27,11 +27,13 @@ The IgxGrid's toolbar is shown using the grid's `showToolbar` property - just se
 </igx-grid>
 ```
 
-#### Exporting
+#### Features
 
-The IgxGrid's toolbar can be configured to allow IgxGrid's data exporting in different formats - MS Excel and CSV. You can enable each exporting format independently by setting its dedicated boolean property to true:
+The IgxGrid's toolbar can be configured to allow columns hiding, columns pinning and exporting data to MS Excel and CSV. You can enable each feature independently by setting its dedicated boolean property to true:
+- for column hiding set `columnHiding` to `true`
+- for column pinning set `columnPinning` to `true`
 - for MS Excel export set the grid's `exportExcel` property to `true`
-- for CSV export set the grid's `exportCsv` property to `true`
+- for CSV export set the grid's `exportCsv` property to `true`.
 
 The available export options are hosted in a drop down which is shown when the main export button on the toolbar is pressed. There are properties for configuring each button's text and they are listed in the API section below.
```

Comment:
We are no longer describing the export options only, so we should either remove or update this paragraph.

Reply:
Done